### ISSUE TITLE
[PD] Working with your team

### ISSUE THREAD

**Issue by UmairRiazDev** [UmairRiazDev/Coursework-Planner#135]
**[<img src="https://avatars.githubusercontent.com/u/42203406?s=17&v=4" width="17" height="17"> @SallyMcGrath](https://github.com/SallyMcGrath)** cloned issue [Migracode-Barcelona/Module-JS1#21](https://github.com/Migracode-Barcelona/Module-JS1/issues/21) on 2024-07-04: 

> ### Coursework content
> 
> 
> 
> Invite one person (fellow trainee or a volunteer) from the class you have worked with this week and arrange a call. Ask for their feedback to help you get more insight into how you are progressing with your learning.
> 
> 
> 
> Write a 250-word essay reflecting on your strengths and development areas as a result of their feedback.
> 
> 
> 
> ### Estimated time in hours
> 
> 
> 
> 2
> 
> 
> 
> ### What is the purpose of this assignment?
> 
> 
> 
> To provide and receive constructive feedback.
> 
> 
> 
> ### How to submit
> 
> 
> 
> Share the link to the Google doc of your reflections on your ticket. Make sure the document is open to view and comment by anyone.

**Comment by UmairRiazDev** [comment]
Reflecting on the feedback, I feel grateful for the positive impact I've had on the team. One strength mentioned is my patience, which helps create an open environment where others feel comfortable asking questions. I’m also happy to hear that my ability to explain complicated ideas in simple terms has made things easier for the team and helped others grow. It’s good to know that my willingness to help and collaborate is appreciated, as I always try to ensure the team succeeds together.

At the same time, I see areas where I can improve. While patience is a strength, I realize I could work on being more assertive, especially when quick decisions are needed. Another area I’d like to develop is delegating tasks better. Sometimes, I tend to take on too much myself, but giving others more responsibility could help them grow and take ownership of their work.

In short, the feedback reminds me of my commitment to being patient, helpful, and supportive. Moving forward, I’ll focus on balancing these qualities with being more assertive when needed and encouraging the team to take on more responsibility.
